### PR TITLE
Update webpack to the lastest release

### PR DESCRIPTION
**Pull request by guillaumevincent**

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -16,18 +16,9 @@ export default () => (
...
-                    presets: ['es2015-webpack']
-                }
-            }, {
-                test: /.json$/,
-                exclude: /node_modules/,
-                use: 'json-loader'
-            }]
-        }
+            rules: [
+                {test: /\.js$/, exclude: /node_modules/, loader: "babel-loader"}
```
Comment (pksjce) [inline webpack.config.babel.js]: Shouldnt this be `use`? And what about the options?

**Comment by guillaumevincent** [comment]
@pksjce in the documentation: https://webpack.js.org/guides/migrating/

```
        loader: "babel-loader", // Do not use "use" here
```

and

`use can be set to a list of loaders`

maybe I'm missing something

File: webpack.config.babel.js (hunk 1 above)
Comment (guillaumevincent) [inline webpack.config.babel.js]: @pksjce in the documentation: https://webpack.js.org/guides/migrating/

```
        loader: "babel-loader", // Do not use "use" here
```

and

`use can be set to a list of loaders`

maybe I'm missing something

So I understand: use `loader` for one loader and `use` if you have more than one loader.
`use` replace `loaderS` > https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules

Comment (pksjce) [inline webpack.config.babel.js]: Oh! Thanks for teaching me that :) 
I only referred to the [schema](https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json) where I think it means both `loader` and `use` can be used interchangeably. Anyway, will try to find out the reasoning. Meanwhile, merging this now. 
Thanks!

Comment (guillaumevincent) [inline webpack.config.babel.js]: thanks

**Pull request merged**